query/filter/comparison.go

```diff
@@ -22,8 +22,11 @@ import (
 )
 
 const (
-	EQ = "$eq"
-	GT = "$gt"
+	EQ  = "$eq"
```

Comment:
should we add `$in` too as discussed earlier?

```
{
 '$in': {
  'state': ['CA', 'FL']
  }
}
```

Reply:
yes, let's do it iteratively. I need to test it on TS as well.